examples/grid.py

```diff
@@ -0,0 +1,59 @@
+import pygame as pg
+
+TITLE = "Grid"
+TILES_HORIZONTAL = 10
+TILES_VERTICAL = 10
+TILE_SIZE = 100
+WINDOW_WIDTH = 1000
+WINDOW_HEIGHT = 1000
+
+
+class Player:
+    def __init__(self, surface):
+        self.surface = surface
+        self.pos = (50, 50)
+
+    def draw(self):
+        pg.draw.circle(self.surface, (255, 255, 255), self.pos, 50)
+
+    def move(self, target):
+        x = (100 * (target[0] // 100)) + 50
+        y = (100 * (target[1] // 100)) + 50

```

Comment:
Is this the same as a math.floor?

Reply:
Yes. Should I use math.floor for syntax reasons?

Reply:
@Masonstiller Yup